hummingbot/templates/conf_avellaneda_market_making_strategy_TEMPLATE.yml

```diff
@@ -44,9 +44,6 @@ add_transaction_costs: null
 # when one side is filled (hanging orders feature) (true/false).
 hanging_orders_enabled: null
 
-## Select way of aggregating hanging orders. Whether if leaving them as they are or calculating a resulting hanging order
```

Comment:
When changing any TEMPLATE.yml it is important to change the "template_version" value (at the beginning of the file)